app/lib/Plugins/InformationService/AAT.php

```diff
@@ -62,7 +62,18 @@ public function __construct() {
 			'label' => _t('Additional sparql suffix'),
 			'description' => _t('Applied after the initial search. Useful to combine filters. For example to limit to children of particular terms enter "?ID gvp:broaderPreferredExtended ?Extended FILTER (?Extended IN (aat:300261086, aat:300264550))"')
 		];
-
+		$g_information_service_settings_AAT['orderBy'] = [
+			'formatType' => FT_TEXT,
+			'display_Type' => DT_FIELD,
+			'default' => '',
+			'width' => 90, 'height' => 1,
+			'label' => _t("Order by"),
+			'description' => _t("How the auto complete list will be ordered. Default is how AAT return the results. Label is by the visible label"),
+			'options' => array(
+				_t("Default") => "Order",
```

Comment:
Maybe make these options configurable in linked_data.conf?

Something like:
```
aat = {
	sortOptions = {
		_(Default) = Order,
		_(Label) = TermPrefLabel,
	},
	...
}
```

But that's not required for this to work just optional.

Reply:
(You'll only be able to read the config after the `parent::__construct()` and it'll be available as:
```
		$va_config = $this->opo_linked_data_conf->get($this->getConfigName());
		$g_information_service_settings_AAT['orderBy']['options'] = $va_config['sortOptions'] ?? [];
```

Reply:
Oops I just saw this was merged ;)